src/helpers/normalize.js

```diff
@@ -2,6 +2,7 @@ const compose = require('compose-then')
 const { result, isPlainObject, curry, get } = require('lodash')
 const pascalCase = require('pascalcase')
 const { downloadEntryImages } = require('./images')
+const { logInfo, logError } = require('./logger')
```

Comment:
Hi, thanks so much for your contribution!

You can remove this import again if you're not using it.

Reply:
I have withdrawn the merge request since this issue has been fixed in the develop branch already.

Reply:
Okay, I'm not sure which version you are currently using, but if you want to get the latest code, you can install with the `next` tag.